src/projectile.cpp

```diff
@@ -0,0 +1,221 @@
+#include "projectile.h"
+
+#include "game.h"
+#include "map.h"
+#include "map_iterator.h"
+#include "explosion.h"
+#include "field.h"
+
+projectile::projectile() :
```

Comment:
Pretty sure pushing it to header would make it necessary to expose the item part of the `unique_ptr<item>`.
Since `damage.h` is all over the place, it shouldn't require `item.h`.